pyproject.toml

```
@@ -75,6 +75,7 @@ pytest = "^7.1"
 pytest-cov = "^3.0"
 pytest-mock = "^3.5"
 pytest-sugar = "^0.9"
+pytest-xdist = "^2.5"
```

Could it be worth including `psutil` extra, as specified in [pytest-xdist documentation](https://pytest-xdist.readthedocs.io/en/latest/index.html#installation)?

```suggestion
pytest-xdist = { version = "^2.5", extras = ["psutil"] }
```

The benefits are not listed in the documentation, but as explained in https://github.com/pytest-dev/pytest-xdist/issues/676, `psutil` is useful for detecting logical CPUs, which may (or may not?) be useful on GitHub Actions.

---

Wasn't sure if it was needed. Will have to dig into a bit more.